<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sys_info_d3d_widget</class>
 <widget class="QWidget" name="sys_info_d3d_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct3D information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Direct3D</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="descriptionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string notr="true"/>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="whatsThis">
             <string notr="true"/>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">S3 Trio 64</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Driver</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="driverLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string notr="true"/>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="whatsThis">
             <string notr="true"/>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">s3trio64_d3d.dll</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Driver version</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="versionLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string notr="true"/>
            </property>
            <property name="statusTip">
             <string notr="true"/>
            </property>
            <property name="whatsThis">
             <string notr="true"/>
            </property>
            <property name="accessibleName">
             <string notr="true"/>
            </property>
            <property name="accessibleDescription">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">66.6</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
